extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -197,6 +198,14 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		return win.innerWidth > win.innerHeight ? '4:3' : '3:4';
 	}
 
+	function getVerticalName(targeting) {
+		var qs = new Querystring();
```

Comment:
It's worth to create this object once (in module), because we call *getPageLevelParams* a few times.